cf_templates/eb_app.yml

```diff
@@ -390,7 +409,40 @@ Resources:
           FromPort: '80'
           ToPort: '80'
           SourceSecurityGroupId: !Ref AWSLBSecurityGroup
-
+  AWSCWRequestActivityMetricFilter:
+    Type: "AWS::Logs::MetricFilter"
+    Properties:
+      LogGroupName: !Join
+        - '/'
+        - - /aws/elasticbeanstalk
+          - !Ref 'AWS::StackName'
+          - var/log/tomcat8/catalina.out
+      FilterPattern: 'org.sagebionetworks.bridge.reporter.worker.BridgeReporterProcessor - Request took'
+      MetricTransformations:
+        -
+          MetricValue: "1"
+          MetricNamespace: "LogMetrics/RequestActivity"
+          MetricName: !Join
+            - '-'
+            - - !Ref 'AWS::StackName'
+              - RequestActivityCount
```

Comment:
Does this alarm automatically write a zero if no log lines are found? Or do we need to put in a default metric value?

Reply:
Yes, I believe it does write 0 if it does not find a match to the FilterPattern.  I think that's what we want?

Reply:
As per https://sagebionetworks.jira.com/browse/BRIDGE-1986, it looks like the DefaultValue parameter is not supported by CloudFormation and needs to be added to the alarm manually.